src/univ/control.cpp

```diff
@@ -85,9 +104,37 @@ void wxControl::UnivDoSetLabel(const wxString& label)
 {
     wxString labelOld = m_label;
     int indexAccelOld = m_indexAccel;
+#if wxUSE_ACCEL
+    wxChar accelCharOld = GetAccelChar();
+#endif // wxUSE_ACCEL
 
     m_indexAccel = FindAccelIndex(label, &m_label);
 
+#if wxUSE_ACCEL
+    wxChar accelChar = GetAccelChar();
+
+    if ( accelCharOld != accelChar )
+    {
+        for ( wxWindow *win = this; win; win = win->GetParent() )
+        {
+            if ( win->IsTopLevel() )
+            {
+                if ( accelCharOld != wxEMPTY_ACCEL_CHAR )
+                {
+                    wxAcceleratorEntry accelEntryOld(wxACCEL_NORMAL, (int)accelCharOld);
+                    win->GetAcceleratorTable()->Remove(accelEntryOld);
+                }
+
+                if ( accelChar != wxEMPTY_ACCEL_CHAR )
+                {
+                    wxAcceleratorEntry accelEntryNew(wxACCEL_NORMAL, (int)accelChar, GetId());
```

Comment:
Shouldn't this be
```suggestion
                    wxAcceleratorEntry accelEntryNew(wxACCEL_ALT, (int)accelChar, GetId());
```
? Mnemonics are supposed to be activated using `Alt-<Letter>`, not just the letter.

Reply:
I used wxACCEL_NORMAL because first I investigated wxMSW builds and noticed there that pressing key without ALT worked fine. But now when I look into these samples I cannot find this case. I can live with wxACCEL_ALT.